src/components/Item/Item.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from 'react';
+import './Item.css';
+import ProgressBar from '../ProgressBar/ProgressBar';
+
+const Item = ({ ItemId, ItemTitle }: {ItemId: number, ItemTitle: string}) => (
```

Comment:
I think that `ItemId`  and `ItemTitle` should be written in camelCase (`itemId` and `itemTitle`